tck/metrics/pom.xml

```diff
@@ -17,13 +17,13 @@
     <modelVersion>4.0.0</modelVersion>
 
     <parent>
-        <groupId>org.eclipse.microprofile.telemetry.tracing</groupId>
-        <artifactId>microprofile-telemetry-tracing-parent</artifactId>
+        <groupId>org.eclipse.microprofile.telemetry</groupId>
+        <artifactId>microprofile-telemetry-tck-parent</artifactId>
         <version>1.2-SNAPSHOT</version>
     </parent>
 
-    <artifactId>microprofile-telemetry-tracing-tck</artifactId>
-    <name>MicroProfile Telemetry Tracing TCK</name>
+    <artifactId>microprofile-telemetry-metrics-tck</artifactId>
+    <name>MicroProfile Telemetry Metrics TCK</name>
 
     <dependencyManagement>
         <dependencies>
```

Comment:
Should the dependency management section be removed since it's in the tck-parent?